<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>playerInfoForm</class>
 <widget class="QWidget" name="playerInfoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>140</width>
    <height>111</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="listWidgetAnswers">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string>QListWidget {
     border: 2px solid gray;
     border-radius: 10px;
	background-color: rgb(255, 255, 150);
}</string>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <item>
    <property name="text">
     <string>arsino</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>oxyphenbutazone</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>senora</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>arseno</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>norias</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>bodega</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>agrafe</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>dagoba</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelUsername">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>triphenylarsine</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAddInfo">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;font color=red&gt;99&lt;/font&gt;</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="avatarLabel" name="labelAvatar">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../client.qrc">:/images/face32.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QToolButton" name="toolButtonSit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Sit</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>avatarLabel</class>
   <extends>QLabel</extends>
   <header>avatarLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../client.qrc"/>
 </resources>
 <connections/>
</ui>
